sample-lnd.conf

```diff
@@ -77,6 +77,16 @@
 ; (with host:port notation), the default port (9735) will be added to the
 ; address.
 ; externalip=
+;
+; Instead of explicitly stating your external IP address, you can also enable
+; UPnP or NAT-PMP support on the daemon. Only one can be selected (either UPnP
+; or NAT-PMP), depending on which one is supported by your network device. This
+; will automatically retrieve your external IP address, even after it has
```

Comment:
Should mention that you need hardware that supports this for it to work.

Reply:
Fixed.